documentation/assemblies/assembly-kafka-connect.adoc

```diff
@@ -32,13 +33,14 @@ m¦FileStreamSinkConnector
 
 |===
 
-In {ProductName}, you can use the Cluster Operator to deploy a Kafka Connect to your Kubernetes cluster.
-OpenShift also supports Kafka Connect Source-2-Image (S2I) clusters.
+You can use the Cluster Operator to deploy a Kafka Connect cluster containing the built-in connectors to your Kubernetes cluster. Deploying a Kafka Connect Source-2-Image (S2I) cluster is supported on OpenShift only. For more information, see xref:using-openshift-s2i-create-image-str[].
 
-A Kafka Connect cluster is implemented as a `Deployment` with a configurable number of workers. The Kafka Connect REST API is available on port 8083, as the `<connect-cluster-name>-connect-api` service.
-
-For more information on deploying a Kafka Connect S2I cluster on OpenShift, see xref:using-openshift-s2i-create-image-str[Creating a container image using OpenShift builds and Source-to-Image].
+A Kafka Connect cluster is implemented as a `Deployment` with a configurable number of workers. The Kafka Connect REST API is available on port 8083 as the `<connect-cluster-name>-connect-api service`. For more information about the supported operations, see the link:https://kafka.apache.org/documentation/#connect_rest[Apache Kafka documentation^].
```

Comment:
should it be `<connect-cluster-name>-connect-api` service and not `<connect-cluster-name>-connect-api service`?

Reply:
You're right. I'll update it.